Rubberduck.Deployment/Rubberduck.Deployment.csproj

```diff
@@ -54,9 +54,7 @@
     <CreateProperty Value="$(ProjectDir)$(OutputPath)$(TargetFileName)">
       <Output TaskParameter="Value" PropertyName="TargetAssembly" />
     </CreateProperty>
-
-    <RubberduckPostBuildTask Config="$(ConfigurationName)" NetToolsDir="$(SdkPath)bin\NETFX 4.6.1 Tools\" WixToolsDir="$(ProjectDir)WixToolset\" SourceDir="$(TargetDir)" TargetDir="$(TargetDir)" ProjectDir="$(ProjectDir)" IncludeDir="$(ProjectDir)InnoSetup\Includes\" FilesToExtract="Rubberduck.dll|Rubberduck.API.dll" />
-
+    <!-- RubberduckPostBuildTask goes here -->
```

Comment:
No, you want to keep it. The last argument, `FilesToExtract` is the one that you need to modify.

```
<RubberduckPostBuildTask 
  Config="$(ConfigurationName)" 
  NetToolsDir="$(SdkPath)bin\NETFX 4.6.1 Tools\" 
  WixToolsDir="$(ProjectDir)WixToolset\" 
  SourceDir="$(TargetDir)" 
  TargetDir="$(TargetDir)" 
  ProjectDir="$(ProjectDir)" 
  IncludeDir="$(ProjectDir)InnoSetup\Includes\" 
  FilesToExtract="Rubberduck.dll" />
```

Reply:
Ah, makes sense, yes